saml-handler/META-INF/NOTICE

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Isn't this handled through the pom mechanism? If I run `mvn clean package` in the code from the Sling Whiteboard, I get a proper notice in the packaged jar.

Reply:
Good catch. This was not intentional. I made a mistake somewhere probably with git rebase